.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   checkout_and_test:
     docker:
       - image: circleci/node:10
-      - image: circleci/mongo:3.6-jessie
+      - image: circleci/mongo:4.4
```

Comment:
I know I asked for this change, but I'm wondering whether we should look at configuring a [matrix build](https://circleci.com/blog/circleci-matrix-jobs/) to test with both 3.6 and 4.4?

Reply:
Good option. Let's add the new step

Reply:
As discussed on Slack, we will create a separate PR to do this CircleCI change

Reply:
Do we need to cache results for mongo3.6 image ?
https://github.com/HPInc/davinci/blob/3e5688651b3510f3707f2ba7414f8d0f58f20b0a/.circleci/config.yml#L35-L37